scss/partials/_menu.scss

```diff
@@ -178,6 +178,17 @@ div.menu {
           border-radius: 4px;
         }
       }
+
+      &.native-app::after {
+        content: "BETA";
+        font-family: $circular_bold;
```

Comment:
@cjsauer please use `OC_Body_Bold()` mixin instead of the circular var directly, this way we can change font w/o having to change var names all around the code.
Check the other usage of that mixin.